src/port1.0/porttrace.tcl

```diff
@@ -213,18 +213,29 @@ namespace eval porttrace {
         }
 
         # Allow access to some Xcode specifics
-        allow trace_sandbox "/var/db/xcode_select_link"
-        allow trace_sandbox "/var/db/mds"
-        allow trace_sandbox [file normalize ~${macportsuser}/Library/Preferences/com.apple.dt.Xcode.plist]
-        allow trace_sandbox "$env(HOME)/Library/Preferences/com.apple.dt.Xcode.plist"
+        set xcode_paths {}
+        lappend xcode_paths "/var/db/xcode_select_link"
+        lappend xcode_paths "/var/db/mds"
+        lappend xcode_paths [file normalize ~${macportsuser}/Library/Preferences/com.apple.dt.Xcode.plist]
+        lappend xcode_paths "$env(HOME)/Library/Preferences/com.apple.dt.Xcode.plist"
 
         # Allow access to developer_dir; however, if it ends with /Contents/Developer, strip
         # that. If it doesn't leave that in place to avoid allowing access to "/"!
         set ddsplit [file split [file normalize [file join ${developer_dir} ".." ".."]]]
         if {[llength $ddsplit] > 2 && [lindex $ddsplit end-1] eq "Contents" && [lindex $ddsplit end] eq "Developer"} {
             set ddsplit [lrange $ddsplit 0 end-2]
         }
-        allow trace_sandbox [file join {*}$ddsplit]
+        lappend xcode_paths [file join {*}$ddsplit]
+
+        if {$use_xcode eq "no"} {
```

Comment:
If use_xcode is a boolean then doing a string comparison is not really correct.

Reply:
Right, I assume because `tbool` should be instead? Changed